From 4316c2659ac8a6437cfa234674d425e0614901a9 Mon Sep 17 00:00:00 2001
From: rofl0r <rofl0r@users.noreply.github.com>
Date: Sun, 12 Jul 2020 16:52:08 +0100
Subject: [PATCH] reassociate(): fix bug

bug was introduced in 488f2e7186d3cddb8df6060d308902986ddb30ac.
the second call to the new function was always meant to use 1 as parameter,
but i typo'ed. thanks to @1yura for the patch.

addressing #303
---
 src/80211.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/80211.c b/src/80211.c
index 92511d4..f976b4c 100644
--- a/src/80211.c
+++ b/src/80211.c
@@ -346,7 +346,7 @@ int reassociate(void)
 				state++;
 				break;
 			case 4:
-				ret = process_authenticate_associate_resp(0);
+				ret = process_authenticate_associate_resp(1);
 				if(ret) state++;
 				else return 0;
 				break;
